src/gate.h

```diff
@@ -1549,13 +1555,13 @@ class custom_gate : public gate
             // operation_type = instr["type"];
             l_attr = "duration";
             duration = instr["duration"];
-#if OPT_USED_HARDWARE
```

Comment:
this define is to identify spots where "used_hardware" is referred to. Leaving it in for now makes it easier to remove this unused code later on

Reply:
moved back